conda/recipes/rapids-build-env/meta.yaml

```diff
@@ -150,8 +150,8 @@ requirements:
     - treelite {{ treelite_version }}
     - twine
     - typing_extensions
-    - ucx-proc=*=gpu
-    - ucx
+    - conda-forge::ucx-proc=*=gpu
+    - conda-forge::ucx {{ ucx_version }}
```

Comment:
Was still seeing `rapidsai-nightly` in [this CI log]( https://gpuci.gpuopenanalytics.com/job/rapidsai/job/integration/job/prb/job/meta-pkg-nightly-build/1028/CONDA_CONFIG_FILE=conda%2Frecipes%2Fversions.yaml,CONDA_USERNAME=rapidsai-nightly,CUDA_VER=11.5,PYTHON_VER=3.9,RAPIDS_VER=22.06.00a/consoleFull ). So added the channel explicitly to hopefully address this.

```
12:36:54     ucx:                           1.12.0+gd367332-cuda11.2_0                   rapidsai-nightly
12:36:54     ucx-proc:                      1.0.0-gpu                                    rapidsai-nightly
```